CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   to screen reader users
 - Improve sale tunnel accessibility, especially when using a keyboard
   or screen reader
+- Fix storybook absolute paths
+- Fix storybook fonts loading
```

Comment:
As this concerns only configuration, we could remove these entries from changelog.

Reply:
But the CI will not pass ? :/